src/python/CRABInterface/RESTWorkerWorkflow.py

```diff
@@ -111,19 +107,19 @@ def fixupTask(task):
 
     result = task._asdict()
 
-    #fixup timestamps
+    # fixup timestamps
     for field in ['tm_start_time', 'tm_start_injection', 'tm_end_injection']:
         current = result[field]
         result[field] = str(getEpochFromDBTime(current)) if current else ''
 
-    #fixup CLOBS values by calling read (only for Oracle)
+    # fixup CLOBS values by calling read (only for Oracle)
     for field in ['tm_task_failure', 'tm_split_args', 'tm_outfiles', 'tm_tfile_outfiles', 'tm_edm_outfiles',
-                  'tm_arguments', 'tm_scriptargs', 'tm_user_files']:
+                  'tm_arguments', 'tm_scriptargs', 'tm_user_files', 'tm_user_config']:
         current = result[field]
         fixedCurr = current if (current is None or isinstance(current, str)) else current.read()
         result[field] = fixedCurr
 
-    #liter_evaluate values
+    # fliter_evaluate values
```

Comment:
the `liter_evaluate` in the comment was actually correct ! please remove leading 'f'

Reply:
I though it flter something...
Fix.